apps/web/src/studio/LocalStudioAuthenticator.tsx

```diff
@@ -75,6 +77,15 @@ export function LocalStudioAuthenticator() {
       return;
     }
 
+    // Wait for environment and apiKeys to be loaded
+    if (!environment || !apiKey) {
+      return;
+    }
+
+    if (environment.name.toLowerCase() !== 'development') {
+      throw new Error('Local Studio works only with development api keys');
+    }
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Lovely!‏